src/Components/SignatureModal/SignatureModal.tsx

```diff
@@ -203,8 +206,11 @@ const styles = StyleSheet.create({
     ...Fonts.style.SubTitle2,
     marginVertical: 5,
   },
+  buttonTextContainer: {
+    marginHorizontal: 4,
```

Comment:
Anyway it is not a way how to fix the bug with the text overflowing the button.

The design of the mobile application needs to be responsive and you should not set the width of the button on the uikit level. The width of the button should be automatically chosen despite the text inside this button.

`minWidth` can be specified to make the buttons not less than something to make it look great. But maximum width should be picked automatically.